src/exports.md

```diff
@@ -2,15 +2,15 @@
 
 The `export` syntax can only be used at the module level, which means it is impossible to have non-deterministic exports. While this can be annoying with collating files that simply import files to only re-export all of their contents, it eliminates whole classes of ambiguity where certain functions, types, etc are only available depending on the state of the runtime environment.
 
-Constants, types, functions, operators, events (and in the future interfaces and classes) are exportable while event handlers are not.
+Constants, types, functions, operators, events, and interfaces are exportable while event handlers are not.
```

Comment:
I think rn when we import another file we bring in all the handlers in that file? That is how the dependency management std works at least

Reply:
We don't bring them in, we just trigger them to be registered but we can't access or call them ourselves in our own module.

I will make note of this particular behavior, though.